index.ts

```diff
@@ -1,68 +1,47 @@
-import {
-  constants as fsConstants,
-  copyFileSync,
-  mkdirSync,
-  readFileSync,
-  rmdirSync,
-  unlinkSync,
-  writeFileSync,
-} from 'fs';
+import { mkdirSync } from 'fs';
 import * as path from 'path';
-import {
-  InputOption,
-  InputOptions,
-  OutputOptions,
-  rollup,
-  RollupBuild,
-  RollupCache,
-} from 'rollup';
-import { heimdall, logger } from './lib/heimdall';
-import OutputPatcher from './lib/output-patcher';
-import Plugin from './lib/plugin';
-import { IRollupOptions } from './lib/rollup';
-import { ITree, treeFromEntries, treeFromPath } from './lib/tree-diff';
+import { InputOptions, OutputOptions } from 'rollup';
+import RollupHelper from './lib/rollup-helper';
 
 // tslint:disable:no-var-requires
 const symlinkOrCopySync: (
   src: string,
   dst: string,
 ) => void = require('symlink-or-copy').sync;
 const nodeModulesPath: (cwd: string) => string = require('node-modules-path');
-// tslint:enable:no-var-requires
 
-const deref =
-  typeof copyFileSync === 'function'
-    ? (srcPath: string, destPath: string) => {
-        try {
-          unlinkSync(destPath);
-        } catch (e) {
-          if (e.code !== 'ENOENT') {
-            throw e;
-          }
-        }
-        copyFileSync(srcPath, destPath, fsConstants.COPYFILE_EXCL);
-      }
-    : (srcPath: string, destPath: string) => {
-        const content = readFileSync(srcPath);
-        writeFileSync(destPath, content);
-      };
+// tslint:disable-next-line:variable-name
+const BroccoliPlugin: new (
```

Comment:
feel free to submit typings to broccoli-plugin, I will accept/merge/release them. :)

Reply:
I think this is done now?